modules/ipi-install-creating-an-rhcos-images-cache.adoc

```diff
@@ -15,6 +15,11 @@ The installation program no longer needs the `clusterOSImage` {op-system} image
 
 If you are running the installation program on a network with limited bandwidth and the {op-system} images download takes more than 15 to 20 minutes, the installation program will timeout. Caching images on a web server will help in such scenarios.
 
+[WARNING]
+====
+If you enable TLS for the HTTPD server, verify the trusted certificate chain between your {product-title} hub and spokes and the HTTPD server. Failure to implement a trusted certificate chain can result in the image creation failing to download. Using untrusted HTTPS servers is not supported.
```

Comment:
I think this should be the same as the warning text in the other file.

Reply:
Updated as suggested.